### PR TITLE
feature(3d-tiles): added support for 3d tileset layer

### PR DESCRIPTION
**Pull request by dotansimha**
@eitanfr @tomermoshe @ttsahi

### CODE REVIEW

File: demo/app/services/app-settings-service/app-settings-service.ts [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
+  private _settings: AppSettings = {
     numberOfEntities: 0,
     entitiesUpdateRate: 0,
     showTracksLayer: true,
     showMapLayer: true,
     realTracksData: false,
     showVelocityVectors: false,
-    showEllipses: false
+    showEllipses: false,
+    show3dtiles: true,
```
Comment (eitanfr) [inline demo/app/services/app-settings-service/app-settings-service.ts]: change the default to false

File: demo/app/services/app-settings-service/app-settings-service.ts [hunk 2]
```diff
@@ -1,20 +1,32 @@
+interface AppSettings {
```
Comment (eitanfr) [inline demo/app/services/app-settings-service/app-settings-service.ts]: nice

File: demo/app/services/app-settings-service/app-settings-service.ts (hunk 1 above)
Comment (dotansimha) [inline demo/app/services/app-settings-service/app-settings-service.ts]: done

File: demo/app/services/app-settings-service/app-settings-service.ts (hunk 2 above)
Comment (dotansimha) [inline demo/app/services/app-settings-service/app-settings-service.ts]: 😘

**Pull request merged**